dist/systemd/system/zincati.service

```diff
@@ -0,0 +1,15 @@
+[Unit]
+Description=Zincati update agent
+Requires=rpm-ostreed.service
+After=rpm-ostreed.service
```

Comment:
I don't think we need these. The daemon isn't long-running; it auto-exits when there are no more clients connected.